Libraries/LibCore/System.cpp

```diff
@@ -793,6 +793,48 @@ ErrorOr<ByteString> current_executable_path()
     size_t len = sizeof(path);
     if (sysctl(mib, 4, path, &len, nullptr, 0) < 0)
```

Comment:
In the commit message: "I also that this is what many other projects" is not worded well